src/csharp/Grpc.Tools/Grpc.Tools.nuspec

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata>
+    <id>Grpc.Tools</id>
+    <title>gRPC C# Tools</title>
+    <summary>Tools for C# implementation of gRPC - an RPC library and framework</summary>
+    <description>Precompiled protobuf compiler and gRPC protobuf compiler plugin for generating gRPC client/server C# code. Binaries are available for Windows, Linux and MacOS.</description>
+    <version>$version$</version>
+    <authors>Google Inc.</authors>
+    <owners>grpc-packages</owners>
+    <licenseUrl>https://github.com/grpc/grpc/blob/master/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/grpc/grpc</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <releaseNotes>Release $version$</releaseNotes>
+    <copyright>Copyright 2015, Google Inc.</copyright>
+    <tags>gRPC RPC Protocol HTTP/2</tags>
+	<dependencies>
+	<dependency id="Grpc" version="$version$" />
+	</dependencies>
+  </metadata>
+  <files>
+    <file src="build\Grpc.Tools.targets" target="build\Grpc.Tools.targets" />
```

Comment:
I'm ok with that approach too, but because Nuget adds a version number to package names, Grpc.Tools.MSBuildIntegration would have to also contain the protoc and grpc_csharp_plugin binaries. This means I'll essentially be replicating Grpc.Tools and adding the build folder (with the targets file) in it.

If that sounds good to you, I can have a pull request ready in a bit. As for being able to customize the command line, I'd like to add this in a subsequent release (I'm trying to think up clean ways to do this).

Do let me know, thanks!